README.md

```diff
@@ -146,6 +146,31 @@ BetoCQ takes three steps to address this issue:
     google.com. Note that in China, this test requires an office VPN network or
     installing a VPN app in devices.
 
+*   **Test host.**
+    The test host should be a Linux or Windows machine with the following
+    libraries installed:
+    - python3.11 or later
+      - Check your Python 3 version number:
+
+      ```
+      python3 --version
+      ```
+
+      - If your version is lower than Python 3.11, install the latest version:
```

Comment:
I'd point them to Python's official site for installation instructions. We don't have to list any cmd here.